lib/saml_idp_constants.rb

```diff
@@ -5,11 +5,6 @@ module Constants
       LOA1_AUTHNCONTEXT_CLASSREF = 'http://idmanagement.gov/ns/assurance/loa/1'.freeze
       LOA3_AUTHNCONTEXT_CLASSREF = 'http://idmanagement.gov/ns/assurance/loa/3'.freeze
       REQUESTED_ATTRIBUTES_CLASSREF = 'http://idmanagement.gov/ns/requested_attributes?ReqAttr='.freeze
-
```

Comment:
The reason why I moved `VALID_AUTHNCONTEXTS` to an initializer is to avoid parsing the JSON in application.yml on every request, which is what we've been doing elsewhere, such as to define `SERVICE_PROVIDERS` and `TWILIO_ACCOUNTS`. By making it a global variable, the JSON gets parsed only once, and the value can be reused.

Is there a better way to do that?